internal/cli/helpers.go

```diff
@@ -0,0 +1,47 @@
+package cli
+
+import (
+	"fmt"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"k8s.io/client-go/util/homedir"
+)
+
+var (
+	defaultClientConfigFile string
+	resourceKinds           = map[string]string{
+		"device":            "",
+		"enrollmentrequest": "",
+		"fleet":             "",
+		"repository":        "",
+		"resourcesync":      "",
+	}
```

Comment:
if you dont expect to use the value you can use map[string]struct{} as a cheaper lookup.
```
resourceKinds := map[string]struct{}{
    "device":            {},
    "enrollmentrequest": {},
    "fleet":             {},
    "repository":       {},
    "resourcesync":     {},
}
```